README.md

```diff
@@ -16,10 +16,32 @@ rich text formatting and natural language due dates.
 - Uses natural language to recognize and highlight due dates
 - Darkmode
 
-## Building locally
+## Usage
+To set tabspace as your new tab page, depending on your browser can either set up the project as a
+browser extension that replaces the new tab page or configure your new tab page
+URL to point to the official tabspace website: https://tabspace.jzhao.xyz/index.html
+
+### Building the browser extension locally
 1. Clone the repository
 2. Install dependencies `npm i`
 3. Build `npm run build`
 4. Load the extension in the Chrome Extensions menu
   1. Go to extensions > Enable developer mode (top right corner)
   2. Click load unpacked, select the `build` folder
+
+For more information, see the [official
+documentation](https://developer.chrome.com/docs/extensions/mv3/getstarted/).
+
+### Setting up the website as your new tab page
+Based on your browser, there may be an option to set the new tab page URL
+directly without installing an extension.
+
+#### Brave
+1. Go to `brave://settings/?search=home` and make sure _Show Home button_ is
+   enabled
+2. Set the homepage to `https://tabspace.jzhao.xyz`
```

Comment:
sorry one more LOL

Reply:
Ah yeah, if you don't mind I have just converted the `.md` file to a `.adoc` file, this way we can make use of variables and store the correct URL once and reference it everywhere in the file